### DISCUSSION TITLE
Guess the song 🎸

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#959] [Galaxy Trivia] [answered]
```
e|------------------------------------------------------------------------18--------|
B|-19-15----19-15----19-15----19-15----19-15------17-14----17-14----17-14-----------|
G|-------16-------16-------16-------16-------16---------15-------15-----------------|
D|----------------------------------------------------------------------------------|
A|----------------------------------------------------------------------------------|
E|----------------------------------------------------------------------------------|
```
**Bonus:** Name the songwriter.

**Comment by CelioAmaral** [comment, marked as answer]
Eagles - Hotel California

Bm
   On a dark desert highway
F#7
    Cool wind in my hair
A
  Warm smell of colitas
E7
   Rising up through the air
G
  Up ahead in the distance
D
  I saw a shimmering light
Em
   My head grew heavy and my sight grew dim
F#7
    I had to stop for the night

[Segunda Estrofe]

Bm
   There she stood in the doorway
F#7
    I heard the mission bell
A
  And I was thinking to myself
E7
   This could be Heaven or this could be Hell
G
  Then she lit up a candle
D
  And she showed me the way
Em
   There were voices down the corridor
F#7
    I thought I heard them say

 

 G                        D
Welcome to the Hotel California
        F#7
Such a lovely place
 Bm
Such a lovely face
  G                              D
Plenty of room at the Hotel California
     Em
Any time of year
         F#7
You can find us here

Segunda parte - repete dedilhado

[Primeira estrofe]

Bm
   Her mind is Tiffany-twisted
F#7
    She got the Mercedes-Benz
A
  She got a lot of pretty, pretty boys
E
  That she calls friends
G
  How they dance in the courtyard
  D
Sweet summer sweat
Em
   Some dance to remember
F#7
    Some dance to forget

[Segunda estrofe]

Bm
   So I called up the Captain
F#7
    Please bring me my wine, he said
A
  We haven't had that spirit here since
E
  Nineteen sixty-nine
G                                          D
  And still those voices are calling from far away
Em
   Wake you up in the middle of the night
F#7
    Just to hear them say

[Refrão]

 G                        D
Welcome to the Hotel California
        F#7
Such a lovely place

 Bm
Such a lovely face
 G                                   D
We're livin' it up at the Hotel California
        Em
What a nice surprise
           F#7
Bring your alibis

[Terceira parte]

[Primeira estrofe - pausa]

Mirrors on the ceiling
The pink champagne on ice, and she said
We are all just prisoners here
Of our own device
And in the master's chambers
They gathered for the feast
The stab it with their steely knives
But they just can't kill the beast

[Segunda estrofe]

Bm
   Last thing I remember, I was
F#7
    Running for the door
A
  I had to find the passage back
         E
To the place I was before
G
  Relax, said the night man
   D
We are programmed to receive
Em
   You can check out any time you like
         F#7
But you can never leave

**Reply by CelioAmaral** [reply]
Songwriting credits go to Don Felder (music), Don Henley, and Glenn Frey (lyrics).

**Reply by ParanoidUser** [reply]
Congratulations! 👏👏 You have just earned the prestigious [Galaxy Brain](https://github.com/CelioAmaral?achievement=galaxy-brain&tab=achievements) badge. Keep up the great work and unlock the Bronze 🥉 badge next by answering 8 questions. Don't forget to post your own questions too! 😊🎉

**Reply by CelioAmaral** [reply]
Thank you, captain! ; D

**Comment by peterbaccaro** [comment]
The song is **Hotel California** by the **Eagles**. It was released in 1976 as the title track of their album "Hotel California." The song has become one of the band's most iconic and enduring hits.

The song Hotel California by the Eagles was written by **Don Felder**, **Don Henley**, and **Glenn Frey**.  Each of the songwriters for "Hotel California" had significant contributions to the song:

- **Don Felder**: Don Felder played the guitar and is credited with the famous guitar riff that opens and drives the song. He also co-wrote the music and collaborated on the overall composition.

- **Don Henley**: Don Henley, the drummer and one of the lead vocalists of the Eagles, contributed to the song's lyrics and melody. He worked on the lyrical content and co-wrote the song with Glenn Frey and Don Felder.

- **Glenn Frey**: Glenn Frey, a guitarist and vocalist in the Eagles, collaborated on the song's lyrics and melody alongside Don Henley and Don Felder. He also provided vocals for the track and played guitar parts.

Together, the three songwriters created the distinct sound and lyrical themes that have made Hotel California one of the most renowned songs in rock music history.

**Comment by thuongtruong1009** [comment]
Indeed, the tablature you provided corresponds to the iconic intro of "Hotel California" by the Eagles. It's a recognizable and popular guitar riff that sets the tone for the song. Well done!
